code/nxsOnto/setup.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+#
+# To use this file type:
+#
+#        python setup.py install
+#
+from setuptools import setup
+
+setup(name='NeXusOntologyGenerator',
+      version='1.1',
+      description='Generates an ontology from nxdl files',
+      author='Steve Collins',
+      url="https://github.com/nexusformat/NeXusOntology/script",
```

Comment:
Does not exist

Reply:
this URL: https://github.com/nexusformat/NeXusOntology/tree/main/script